Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'active_model_serializers'
 gem 'omniauth-google-oauth2'
 gem 'will_paginate'
 gem 'acts-as-taggable-on', '~> 6.0'
-gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
+# gem 'omniauth-census', git: "https://github.com/turingschool-projects/omniauth-census"
+gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'
+# gem 'devise'
```

Comment:
Took out those two gems as they were not needed. I think I was getting an error due to the first omniauth gem.